Docs status updates - Fixes #150 (#268)
nautobot/nautobot: nautobot/docs/models/virtualization/virtualmachine.md

A virtual machine represents a virtual compute instance hosted within a cluster. Each VM must be assigned to exactly one cluster.

> Like devices, each VM can be assigned a platform and/or functional role, and must have one of the following operational [`statuses`](https://nautobot.readthedocs.io/en/latest/models/extras/status/) assigned to it:

Reviewer: ```suggestion
Like devices, each VM can be assigned a platform and/or functional role, and an operational [`status`](https://nautobot.readthedocs.io/en/latest/models/extras/status/). The following statuses are available by default:
```

* Failed
* Decommissioning

> Status fields can now be customized via the new [`status`](https://nautobot.readthedocs.io/en/latest/models/extras/status/) model.

Reviewer: Same: IMO just remove this since it's redundant with the link above it.
